src/www/app/app.ts

```diff
@@ -153,83 +153,91 @@ export class App {
     let buttonLink: string;
 
-    if (e instanceof errors.VpnPermissionNotGranted) {
-      messageKey = 'outline-plugin-error-vpn-permission-not-granted';
-    } else if (e instanceof errors.InvalidServerCredentials) {
-      messageKey = 'outline-plugin-error-invalid-server-credentials';
-    } else if (e instanceof errors.RemoteUdpForwardingDisabled) {
-      messageKey = 'outline-plugin-error-udp-forwarding-not-enabled';
-    } else if (e instanceof errors.ServerUnreachable) {
-      messageKey = 'outline-plugin-error-server-unreachable';
-    } else if (e instanceof errors.FeedbackSubmissionError) {
-      messageKey = 'error-feedback-submission';
-    } else if (e instanceof errors.ServerUrlInvalid) {
-      messageKey = 'error-invalid-access-key';
-    } else if (e instanceof errors.ServerIncompatible) {
-      messageKey = 'error-server-incompatible';
-    } else if (e instanceof OperationTimedOut) {
-      messageKey = 'error-timeout';
-    } else if (e instanceof errors.ShadowsocksStartFailure && this.isWindows()) {
+    if (error instanceof errors.VpnPermissionNotGranted) {
+      toastMessage = this.localize('outline-plugin-error-vpn-permission-not-granted');
+    } else if (error instanceof errors.InvalidServerCredentials) {
+      toastMessage = this.localize('outline-plugin-error-invalid-server-credentials');
+    } else if (error instanceof errors.RemoteUdpForwardingDisabled) {
+      toastMessage = this.localize('outline-plugin-error-udp-forwarding-not-enabled');
+    } else if (error instanceof errors.ServerUnreachable) {
+      toastMessage = this.localize('outline-plugin-error-server-unreachable');
+    } else if (error instanceof errors.FeedbackSubmissionError) {
+      toastMessage = this.localize('error-feedback-submission');
+    } else if (error instanceof errors.ServerUrlInvalid) {
+      toastMessage = this.localize('error-invalid-access-key');
+    } else if (error instanceof errors.ServerIncompatible) {
+      toastMessage = this.localize('error-server-incompatible');
+    } else if (error instanceof OperationTimedOut) {
+      toastMessage = this.localize('error-timeout');
+    } else if (error instanceof errors.ShadowsocksStartFailure && this.isWindows()) {
       // Fall through to `error-unexpected` for other platforms.
-      messageKey = 'outline-plugin-error-antivirus';
-      buttonKey = 'fix-this';
+      toastMessage = this.localize('outline-plugin-error-antivirus');
+      buttonMessage = this.localize('fix-this');
       buttonLink = 'https://s3.amazonaws.com/outline-vpn/index.html#/en/support/antivirusBlock';
-    } else if (e instanceof errors.ConfigureSystemProxyFailure) {
-      messageKey = 'outline-plugin-error-routing-tables';
-      buttonKey = 'feedback-page-title';
+    } else if (error instanceof errors.ConfigureSystemProxyFailure) {
+      toastMessage = this.localize('outline-plugin-error-routing-tables');
+      buttonMessage = this.localize('feedback-page-title');
```

Comment:
Not for this PR, but I wonder if we should omit the link to the feedback form...